docs/docs/apis/observability/metrics.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Metrics
```

Comment:
@fforootd is there nothing available on ZITADEL Cloud?
Would it be useful to mention the HealthZ endpoints on the APIs?

Reply:
Yes, the cloud does not expose the endpoint at the moment, but we could make exceptions if needed.

Healthz and Readiness I can append something.

Reply:
That would be great! Just as a pointer / hint.

Reply:
Its done!

Reply:
https://github.com/zitadel/zitadel/pull/4376/files#diff-b7564c95acfd8781351c5ae69d9dd403deb0ac925282ab0e50b2e8c307c47abc